samples/sample-manifest.json

```diff
@@ -20,13 +20,24 @@
             "MaxActivations": 1,
             "Parameters": {
                 "Files": {
-                    "/tmp/defg.txt": "foo",
-                    "/tmp/jkl.mno": "bar"
+                    "/tmp/abc.txt": "helloworld",
+                    "/tmp/helloworld.sh": "#!/usr/bin/env bash\necho {{ raw .Secrets.hello }} {{ raw .Secrets.hello }}",
+                    "/tmp/defg.txt": {
+                        "encoding": "string",
+                        "data": "foo"
+                    },
+                    "/tmp/jkl.mno": {
+                        "encoding": "base64",
```

Comment:
I agree and would go for another type

Reply:
Would like to have this as a separate Flag or directly in the encoding type?
```
"/tmp/jkl.mno": {
   "encoding": "base64",
   "templates": false,
...
```
```
"/tmp/jkl.mno": {
   "encoding": "base64withoutTemplates",
...
"/tmp/jkl.mno": {
   "encoding": "base64withTemplates",
```

Reply:
I would go for directly having it as part of the encoding type.
Since there will be more encodings than base64, having an additional flag that only affects the one encoding can be confusing.

Reply:
The flag would affect all encoding types and we would only need half the encoding types though.

Reply:
Ah, i might have misunderstood how the encoding field was to be used.
In that case I agree that an additional flag would be preferable to save on additional encoding types.

But do we actually need a flag to showcase the data using templates?
From the Coordinators point of view we can easily check if there are any template placeholders.
It would only really be beneficial for people reading the manifest, in which case it might be more helpful to have something like a `comment` field, where users can add a short text.
This field would not need be evaluated by the Coordinator in any way, but simply serve as a way to communicate the purpose and/or content of the data to other users.

Reply:
After the short discussion today we decided to go with two flags: "encoding" and "noTemplates".
This allows for simpler adding of future encodings as for example "hex".